src/client/eppo-client.ts

```diff
@@ -128,32 +129,27 @@ export default class EppoClient {
         'Eppo SDK unable to fetch flag configurations without configuration request parameters',
       );
     }
+    // if fetchFlagConfigurations() was previously called, stop any polling process from that call
+    this.requestPoller?.stop();
 
-    if (this.requestPoller) {
-      // if fetchFlagConfigurations() was previously called, stop any polling process from that call
-      this.requestPoller.stop();
-    }
-
-    const isExpired = await this.flagConfigurationStore.isExpired();
-    if (!isExpired) {
-      logger.info(
-        '[Eppo SDK] Configuration store is not expired. Skipping fetching flag configurations',
-      );
-      return;
-    }
     const {
       apiKey,
       sdkName,
       sdkVersion,
       baseUrl, // Default is set in ApiEndpoints constructor if undefined
       requestTimeoutMs = DEFAULT_REQUEST_TIMEOUT_MS,
+      pollingIntervalMs = DEFAULT_POLL_INTERVAL_MS,
       numInitialRequestRetries = DEFAULT_INITIAL_CONFIG_REQUEST_RETRIES,
       numPollRequestRetries = DEFAULT_POLL_CONFIG_REQUEST_RETRIES,
       pollAfterSuccessfulInitialization = false,
       pollAfterFailedInitialization = false,
       throwOnFailedInitialization = false,
       skipInitialPoll = false,
     } = this.configurationRequestParameters;
+    if (pollingIntervalMs <= 0) {
```

Comment:
What's a reasonable guard/error-action to take here?

Reply:
I think we can still log the error, and then perhaps set it to the default? Alternatively, we could use an invalid polling interval to mean don't poll, which would be consistent with other SDKs

Reply:
I think setting to default is the way to go. Since there are `pollAfterSuccessfulInitialization` and `pollAfterFailedInitialization`, it doesn't make sense for an invalid argument to implicitly override the variables explicitly controlling this behaviour.